main.py

```diff
@@ -1,15 +1,32 @@
-import sys
+import os, sys
+from uni_henn.utils.context import Context
 
 if __name__ == "__main__":
     if len(sys.argv) < 2:
         print("Please input argv")
         exit(0)
 
+    """
+    When creating a context, you can adjust several parameters.
+    @param N        Number of slots                         (default: 8192)
+    @param depth    Depth of ciphertext                     (default: 11)
+    @param LogQ     Fraction log scale                      (default: 32)
+    @param LogP     Fraction log scale + Integer log scale  (default: 40)
+    @param scale    Scale value used in ecoding             (default: 2**LogQ)
+    """
+    context = Context(N = 2**14, depth = 8, LogQ = 40, LogP = 60)
+    # context = Context()
+    
     files = {'1': 'example/M1_test.py',
             '2': 'example/M2_test.py',
             '3': 'example/M3_test.py',
             '4': 'example/M4_test.py',
             '5': 'example/M5_test.py',
             '6': 'example/M6_test.py',}
+    
+    TestFile = open(files[sys.argv[1]])
+    TestCode = TestFile.read()
 
-    exec(open(files[sys.argv[1]]).read())
\ No newline at end of file
+    sys.argv = [files[sys.argv[1]], context]
+    exec(TestCode)
+    TestFile.close()
```

Comment:
이러면 리소스 관리를 with에서 해주기 때문에 close가 필요없게 됩니다
```suggestion
    sys.argv = [test_filename], context]
    exec(test_code)
```

Reply:
조언 감사드립니다!
말씀해주신 대로 argv를 int로 바꾸고 1~6인지 확인하는 exception code와 with을 사용하는 방식으로 변경했는데, 혹시 현재 코드이면 괜찮나요??

Reply:
좋습니다 👍

Reply:
확인 감사드립니다! 
PR 재요청 드렸는데 다시 approve 해주실 수 있을까요?